src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -7183,5 +7183,714 @@ static void F()
                     generation0,
                     ImmutableArray.Create(new SemanticEdit(SemanticEditKind.Update, f0, f1, GetSyntaxMapFromMarkers(source0, source1), preserveLocalVariables: true))));
         }
+
+        [Fact]
+        public void OutVar()
+        {
+            var source0 = MarkedSource(@"
+class C
+{
+    static void F(out int x, out int y) { x = 1; y = 2; }
+    static int G() { F(out int x, out var y); return x + y; }
```

Comment:
>x [](start = 31, length = 1)

You need to mark the declarators that match:
In source0: ```<N:0>x</N:0>``` 
In source1: ```<N:0>x</N:0>``` 
In source2: ```<N:0>x</N:0>```